src/runtime/node/internal/util/types.ts

```
@@ -1,72 +1,92 @@
-  typeof nodeUtilTypes.isArgumentsObject
->("util.types.isArgumentsObject");
+export const isArgumentsObject: typeof nodeUtilTypes.isArgumentsObject = (
+  val: unknown,
+): val is IArguments =>
+  Object.prototype.toString.call(val) === "[object Arguments]";
 
-export const isBigIntObject = (val: any): val is bigint =>
-  typeof val === "bigint";
+export const isBigIntObject: typeof nodeUtilTypes.isBigIntObject = (
+  val: unknown,
+): val is BigInt => val instanceof BigInt;
```

ESLint

[error] Prefer using the primitive `bigint` as a type name, rather than the upper-cased `BigInt`.

(`@typescript-eslint/no-wrapper-object-types`)

---

[error] Avoid using `instanceof` for type checking as it can lead to unreliable results.

(unicorn/no-instanceof-builtins)